scripts/ci/ci_check_license.sh

```diff
@@ -22,21 +22,39 @@ export PYTHON_VERSION=${PYTHON_VERSION:-3.6}
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
 function run_check_license() {
-    docker run "${EXTRA_DOCKER_FLAGS[@]}" -t \
-            --entrypoint "/usr/local/bin/dumb-init"  \
-            --env PYTHONDONTWRITEBYTECODE \
-            --env VERBOSE \
-            --env VERBOSE_COMMANDS \
-            --env HOST_USER_ID="$(id -ur)" \
-            --env HOST_GROUP_ID="$(id -gr)" \
-            --rm \
-            "${AIRFLOW_CI_IMAGE}" \
-            "--" "/opt/airflow/scripts/ci/in_container/run_check_licence.sh" \
-            | tee -a "${OUTPUT_LOG}"
-}
+    echo
+    echo "Running Licence check"
+    echo
+
+    # This is the target of a symlink in airflow/www/static/docs -
+    # and rat exclude doesn't cope with the symlink target doesn't exist
+    mkdir -p docs/_build/html/
 
-prepare_build
+    echo "Running license checks. This can take a while."
 
-rebuild_ci_image_if_needed
+    if ! docker run "${EXTRA_DOCKER_FLAGS[@]}" -t \
+            --user "$(id -ur):$(id -gr)" \
+            --rm \
+            ashb/apache-rat:0.13-1 \
```

Comment:
Have you ever asked the application developers if they didn't want to start publishing an official image?

Reply:
I'm not really sure which project it's part of, it seems like a side project?

Reply:
This is the official Apache project. Here is the Github repository:  https://github.com/apache/creadur-rat/
I've already added Docker to Travis CI, so the Docker image is an interesting next step forward.
https://github.com/apache/creadur-rat/blob/master/.travis.yml